docs/TRANSLATING.md

```diff
@@ -246,6 +262,39 @@ git push --force # Push the changes to your repo. Be aware that this will overwr
 
 ---
 
+## Adding a new language (checklist)
+
+The following section describes how to add a new language and what to do or wait for.
+Please open an issue on GitHub and introduce yourself first, even if you intend to add the translation files yourself.
+
+After having opened an issue on GitHub:
+1. Fork the Jamulus repository as described above, then install Qt on your machine (See [COMPILING.md](../COMPILING.md)). (You will need `lrelease` and `lupdate` later on.)
+2. Create a branch based on the `master` branch: Locally navigate to the folder you cloned the repo (with `cd /path/to/jamulus/repo`) and type `git checkout -b "translate-add-xx_YY"` where xx_YY is the language code of the new language.
```

Comment:
```suggestion
2. Create a branch based on the `master` branch: Locally navigate to the folder you cloned the repo to (with `cd /path/to/jamulus/repo`) and type `git checkout -b "translate-add-xx_YY"` where xx_YY is the language code of the new language.
```

Reply:
We use repo and repository interchangeably as mentioned in the git section.

Reply:
I try to make it "repository", because it at best sells people on an unfamiliar concept.
"Repository" is a little easier to understand than "repo" in that sense.